webpack.config.js

```diff
@@ -1,12 +1,11 @@
+/* eslint-disable no-var */
+var webpack = require('webpack');
+var production = process.env.NODE_ENV === 'production';
```

Comment:
Instead, check the `-p` flag using `process.argv`. No need of extra variable this way.